src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests3.cs

```diff
@@ -5501,5 +5501,107 @@ static void Main()
                 );
             var compVerifier = CompileAndVerify(compilation, expectedOutput: expectedOutput);
         }
+
+        [Fact, WorkItem(44518, "https://github.com/dotnet/roslyn/issues/44518")]
+        public void Crash_01()
```

Comment:
>Crash_01 [](start = 20, length = 8)

Minor: Consider naming these something like "InvalidConstant_01", etc. instead since presumably the tests shouldn't crash.